server/tests/metadata.rs

```diff
@@ -1,28 +1,37 @@
 mod entities_metadata_query {
     use chrono::{DateTime, Duration, Utc};
+    use itertools::Itertools;
     use std::time::{SystemTime, UNIX_EPOCH};
     use test_utils::{assert_eq, *};
 
+    fn check_and_get_dates(
+        json_response: &serde_json::Value,
+    ) -> (DateTime<chrono::FixedOffset>, DateTime<chrono::FixedOffset>) {
+        ["dateCreated", "dateModified"]
+            .into_iter()
+            .map(|key| {
+                DateTime::parse_from_rfc3339(
+                    &json_response["entities"][0]["mainEntityOfPage"][key]
+                        .as_str()
+                        .unwrap(),
+                )
+                .unwrap()
+            })
+            .map(|date_from_response| {
+                assert!(Utc::now() < date_from_response + Duration::seconds(1));
+                date_from_response
+            })
+            .collect_tuple()
+            .unwrap()
+    }
+
     #[actix_rt::test]
     async fn returns_metadata_for_articles() {
         Message::new("EntitiesMetadataQuery", json!({ "first": 1 }))
             .execute()
             .await
             .should_be_ok_with(|actual_response| {
-        let actual_date_created = DateTime::parse_from_rfc3339(
-          &actual_response["entities"][0]["mainEntityOfPage"]["dateCreated"]
-            .as_str()
-            .unwrap()
-        ).unwrap();
-        assert!(Utc::now() < actual_date_created + Duration::seconds(1));
-
-        let actual_date_modified = DateTime::parse_from_rfc3339(
-          &actual_response["entities"][0]["mainEntityOfPage"]["dateCreated"]
-            .as_str()
-            .unwrap()
-        ).unwrap();
-        assert!(Utc::now() < actual_date_modified + Duration::seconds(1));
-
+        let (actual_date_created, actual_date_modified) = check_and_get_dates(&actual_response);
```

Comment:
The indentation doesn't look right here and different than in similar cases below. 
It still looks the same after format though, any idea or tip @kulla ?

Reply:
Yes, I have no idea why this actually happens -> in src/metadata.rs we have a similar problem. I think it might have to do with using macros...

Reply:
Indeed, macro declarations and uses is mentioned under [rustfmt limiatations](https://github.com/rust-lang/rustfmt#limitations). Now we know.